sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandTestUtils.scala

```diff
@@ -91,4 +91,20 @@ trait DDLCommandTestUtils extends SQLTestUtils {
   }
 
   protected def checkLocation(t: String, spec: TablePartitionSpec, expected: String): Unit
+
+  // Getting the total table size in the filesystem in bytes
+  def getTableSize(t: String): Int = {
```

Comment:
nit: Should we use `Long` as return type? yeah I know it's supposed to be only in test but just to be on the safe side.

Also perhaps rename `t` to `tableName` or something so it is easier to read. :)

Reply:
We discussed this with @cloud-fan in other PRs, and decided to use short names for variables with small scope.

If you look at the unified tests, you can see `t` everywhere used for table names. This made sql statements shorter.